Sources/KSCrashRecording/Monitors/KSCrashMonitor_MachException.c

```diff
@@ -591,7 +592,10 @@ static void addContextualInfoToEvent(struct KSCrash_MonitorContext *eventContext
 {
     const char *signalName = kscm_signal_getAPI()->monitorId();
 
-    if (signalName && strcmp(eventContext->monitorId, signalName) == 0) {
+    if (strcmp(eventContext->monitorId, kscm_watchdog_getAPI()->monitorId()) == 0) {
```

Comment:
[nit] Because of that, we’re adding more and more coupling between the monitors. Same for signal